CHANGES.md

```diff
@@ -75,6 +75,8 @@ To be released.
     messages when a `Swarm` instance is requested to terminate.
  -  Added `NamespaceNotFoundException` class.  [[#232]]
  -  Added `Block<T>.Evaluate()` method.  [[#243]]
+ -  Made `InvalidBlockTimestampException` class `public` so that it can be
+    caught, [[#133], [#251]]
```

Comment:
Could you replace this comma with a period?  “caught,” → “caught.”

Reply:
Oh, I missed typos  
I will fix it